components/CalendlyComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+
+const CalendlyComponent: React.FC = () => {
+	useEffect(() => {
+		const head = document.querySelector("head");
+		const script = document.createElement("script");
+		script.setAttribute(
+			"src",
+			"https://assets.calendly.com/assets/external/widget.js"
+		);
+		head?.appendChild(script);
+	}, []);
```

Comment:
this feels suss rather than re-using an existing Calendly component off the shelf

Reply:
I figured it would look like that but this is how I managed to make it work so I stuck with it. 

"if it aint broke dont fix it" mentality if you will haha.